c/src/ml-api-inference-single.c

```diff
@@ -1083,6 +1091,21 @@ ml_single_open_custom (ml_single_h * single, ml_single_preset * info)
     goto error;
   }
 
+  /* handle flexible single */
+  /**
+   * Set invoke_dynamic as TRUE if the given nnfw do invoke_dynamic
+   *
+   * if (info->nnfw == ML_NNFW_TYPE_EXECUTORCH_LLAMA || info->nnfw == ML_NNFW_TYPE_LLAMACPP) {
+   *   invoke_dynamic = TRUE;
```

Comment:
It should be set by the subplugin or pipeline user, not by the API implementation.
The API implementation (and nnstreamer core) should NOT aware such details.